cmd/crc/cmd/root.go

```diff
@@ -169,13 +169,14 @@ func addForceFlag(cmd *cobra.Command) {
 
 func executeWithLogging(fullCmd string, input func(cmd *cobra.Command, args []string) error) func(cmd *cobra.Command, args []string) error {
 	return func(cmd *cobra.Command, args []string) error {
-		if err := input(cmd, args); err != nil {
+		logging.Debugf("Command '%s' started", fullCmd)
+		err := input(cmd, args)
+		if err != nil {
 			if serr := segmentClient.Upload(fullCmd, err); serr != nil {
-				fmt.Println(serr.Error())
+				logging.Debugf("Cannot send data to telemetry", serr.Error())
 			}
-			return err
 		}
-		return nil
+		return err
```

Comment:
This is doing more than what the log says, I'd expect a one line commit given the log.

Reply:
I simplified and stuck to the original purpose of the PR :) I removed this.